src/Sulu/Bundle/MediaBundle/DependencyInjection/Configuration.php

```diff
@@ -57,7 +57,11 @@ public function getConfigTreeBuilder()
                 ->addDefaultsIfNotSet()
                 ->children()
                     ->arrayNode('response_headers')
-                        ->prototype('scalar')->end()
+                        ->prototype('scalar')->end()->defaultValue(array(
+                            'Expires' => '+1 month',
+                            'Pragma' => 'public',
+                            'Cache-Control' => 'public'
```

Comment:
yes I set them as the default value that we can delete them from the standard installation process: https://github.com/sulu-io/sulu-standard/pull/437/files